modules/ts/src/ts_perf.cpp

```diff
@@ -665,11 +669,13 @@ static int perf_validation_idle_delay_ms = 3000; // 3 sec
 static void loadPerfValidationResults(const std::string& fileName)
 {
     perf_validation_results.clear();
+
+    float value = 0;
+#ifndef WINRT
     std::ifstream infile(fileName.c_str());
```

Comment:
Please find detailed description of reasons of these updates [there](https://github.com/Itseez/opencv/pull/4064#issuecomment-107431788)

I can suggest placing of full function body into `#ifdef` blocks there for now. This supposes some duplication of code however separates WinRT and non-WinRT code to make this part more maintainable.